test/test_jit.py

```diff
@@ -1,91 +1,132 @@
 # -*- coding: utf-8 -*-
 # Owner(s): ["oncall: jit"]
 
-import torch
+import copy
```

Comment:
Remove any formatting changes so it's clear what the semantic changes are.
(Code outside of the onnx directories is not consistently formatted, and in general you should not auto-format it when modifying it).